validation/src/main/scala/hmda/validation/rules/IfInstitutionPresentIn.scala

```diff
@@ -0,0 +1,19 @@
+package hmda.validation.rules
+
+import hmda.model.institution.Institution
+import hmda.validation.context.ValidationContext
+import hmda.validation.dsl.{ Result, Success }
+
+object IfInstitutionPresentIn {
+  def apply[T](ctx: ValidationContext)(constructor: (Institution) => EditCheck[T]): EditCheck[T] = {
+    ctx.institution match {
+      case Some(inst) => constructor(inst)
+      case None => new EmptyEditCheck
+    }
+  }
+}
+
+class EmptyEditCheck[T] extends EditCheck[T] {
```

Comment:
Good point. Those options both seemed good, but IntelliJ made it easier to do the former, so I went with it.
Should be ready now!